packages/plugin-ext/src/plugin/node/debug/debug.ts

```diff
@@ -184,6 +187,31 @@ export class DebugExtImpl implements DebugExt {
         return Disposable.create(() => this.descriptorFactories.delete(debugType));
     }
 
+    static SCHEME = 'debug';
+    static SCHEME_PATTERN = /^[a-zA-Z][a-zA-Z0-9\+\-\.]+:/;
```

Comment:
minor: move the `static` members to the top of the class.

Reply:
It also appears that both the scheme and the pattern are already declared in `debug-source.ts` as `DebugSource.SCHEME` and `DebugSource.SCHEME_PATTERN`.  Would it be possible to reuse those declarations?